docs/swarm/join-nodes.md

```diff
@@ -0,0 +1,131 @@
+<!--[metadata]>
++++
+title = "Join nodes to a swarm"
+description = "Add worker and manager nodes to a swarm"
+keywords = ["guide, swarm mode, node"]
+advisory = "rc"
+[menu.main]
+identifier="join-nodes-guide"
+parent="engine_swarm"
+weight=13
++++
+<![end-metadata]-->
+
+# Join nodes to a swarm
+
+When you first create a swarm, you place a single Docker Engine (Engine) into
+swarm mode. To take full advantage of swarm mode you can add nodes to the swarm:
+
+* Adding worker nodes increase capacity. When you deploy a service to a swarm,
+the Engine schedules the tasks on worker nodes. When you add workers to your
+swarm, you increase the scale of the swarm to handle tasks.
+* Manager nodes increase fault-tolerance. Manager nodes perform the
+orchestration and cluster management functions for the swarm. Among manager
+nodes, a single leader node conducts orchestration tasks. If a leader node
+goes down, the remainin manager nodes will elect a new leader and resume
+orchestration and maintenance of the swarm state.
+
+Before you add nodes to a swarm you must install Docker Engine 1.12 or later on
+the host machine.
+
+See "View swarm configurations" on [Run Docker Engine in swarm mode](swarm-mode.md/#view-swarm-configurations) for more information.
+
+
+## Join as a worker node
+
+To retrieve the join command including the join token for worker nodes, run the
+following command on a manager node:
+
+```bash
+$ docker swarm join-token worker
+
```

Comment:
We've been changing some examples to add a blank line between the command that's entered, and the output. It's not matching the actual output, but we got feedback that it's sometimes unclear what the user should *type* and what is output. Adding an extra line helps making it slightly more clear